### DISCUSSION TITLE
the cpu is only used at 3.5% and the server lags

### DISCUSSION THREAD

**Discussion by coolsl** [DarkflameUniverse/DarkflameServer#1526] [Q&A] [answered]
hey, in the 2nd zone after the rocket the server starts to lag but when I do htop to have the performance the cpu is only used at 3.5% is there a way to increase the use of the cpu? the ram is used at 500MB/1GB

**Comment by aronwk-aaron** [comment, marked as answer]
in your `sharedconfig.ini` set the bandwidth limit to 0

**Reply by coolsl** [reply]
thank you it works now ^^
